core/src/main/java/org/geysermc/geyser/registry/populator/ItemRegistryPopulator.java

```diff
@@ -210,8 +212,7 @@ public static void populate() {
 
             Set<String> javaOnlyItems = new ObjectOpenHashSet<>();
             Collections.addAll(javaOnlyItems, "minecraft:spectral_arrow", "minecraft:debug_stick",
-                    "minecraft:knowledge_book", "minecraft:tipped_arrow", "minecraft:trader_llama_spawn_egg",
```

Comment:
The trader llama spawn egg still only exists on Java in 1.19.0 so this still needs to be done. The `additionalTranslatedItems` field in PaletteVersion should cover this just for that version.